frontend/device/remarkable/device.lua

```diff
@@ -130,6 +134,32 @@ end
 
 function Remarkable:supportsScreensaver() return true end
 
+function Remarkable2:initNetworkManager(NetworkMgr)
```

Comment:
I think we can use the same network manager for both remarkables: will add a couple of comments

Reply:
on top of `initNetworkManager` do

```lua
local model = self.model
```